app/views/docs/migrations-firebase.phtml

```diff
@@ -0,0 +1,66 @@
+<p>
```

Comment:
Please make sure to merge my PR on firebase to add more information Project keys

Reply:
Added